lib/util/crashlytics_handler.dart

```diff
@@ -0,0 +1,9 @@
+import 'package:firebase_crashlytics/firebase_crashlytics.dart';
+import 'package:flutter/material.dart';
+
+class FireBaseCrashLytics {
```

Comment:
Rename `FireBaseCrashLytics` to `FirebaseCrashLytics` since Fireabase is a single word product name.